app/views/reports/standard_form_report/subset/_reference.html.erb

```diff
@@ -0,0 +1,7 @@
+<% if @report.references? %>
+  <td class="reference">
+    <% if summary.questioning.reference.present? %>
```

Comment:
Hooray for splitting up files! This is a nice little partial!

One issue is you have an LoD violation here. The way to fix it is to add a delegator to summary that delegates `reference` to `questioning` (no prefix needed here I think).

Reply:
done!